### DISCUSSION TITLE
CLion open source licenses

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3881] [General]
As you might have noticed, in the bottom of our README file of the main repo, we have listed [JetBrains](https://www.jetbrains.com/?from=AzerothCore) as a sponsor of AzerothCore.

JetBrains is a company that develops cross-platform IDEs for many different languages like IntelliJ, WebStorm, PHPStorm, PyCharm, etc...

For C++ they provide [CLion](https://www.jetbrains.com/clion/). Normally you can use CLion for free for 1 month trial period, then you have to pay for it.

**Jetbrains has given AzerothCore free open source licenses to use CLion**. This means that all official contributors of AzerothCore are given a free, unlimited license to use it in order to develop AzerothCore.

**For AC-contributors only**: if you need a CLion license (to use **ONLY** to develop AC), please send me a PM and I will grant you one.

You may find this useful:

https://stackoverflow.com/questions/54206799/how-to-run-and-debug-azerothcore-using-clion

**Comment by BarbzYHOOL** [comment]
So we can have one for any (active?) team member of AzerothCore?

**Comment by FrancescoBorzi** [comment]
The amount of licenses is limited, but as we keep growing I can ask for more licenses.
